src/api/v1/middlewares/validation/index.js

```diff
@@ -1 +1 @@
-export * from './userSignup';
\ No newline at end of file
+export * from "./userSignup.js";
```

Comment:
you can delete this index.js file. We don't need it. We'd import the validation modules directly in the router files.